test/test_api/test_api.py

```diff
@@ -714,3 +714,61 @@ def test_build_pipeline(api_type, fit_dictionary_tabular):
     pipeline = api.build_pipeline(fit_dictionary_tabular['dataset_properties'])
     assert isinstance(pipeline, BaseEstimator)
     assert len(pipeline.steps) > 0
+
+
+@unittest.mock.patch('autoPyTorch.evaluation.train_evaluator.eval_function',
+                     new=dummy_eval_function)
+@pytest.mark.parametrize('dataset_name', ('iris',))
+def test_fit_ensemble(backend, n_samples, dataset_name):
```

Comment:
Can you also check that the ensemble history got populated correctly with at least one run?